### ISSUE TITLE
aursync: couldn't read spotify/.git/packed-refs: Not a directory

### ISSUE THREAD

**Issue by jendrikw** [AladW/aurutils#332]
I did a aursync -u when the following happened:
```
  -> Using [custom] repository
==> Resolving dependencies...
  -> spotify 1.0.72.117-1 -> 1.0.77.336-1
==> Retrieving build files...
POST git-upload-pack (972 bytes)
remote: Counting objects: 4, done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4 (delta 3), reused 0 (delta 0)
Unpacking objects: 100% (4/4), done.
From https://aur.archlinux.org/spotify
   b18e792..dda96cb  master     -> origin/master
fatal: couldn't read spotify/.git/packed-refs: Not a directory
```

Simply trying again, I got this:
```
  -> Using [custom] repository
==> Resolving dependencies...
  -> spotify 1.0.72.117-1 -> 1.0.77.336-1
==> Retrieving build files...
From https://aur.archlinux.org/spotify
 = [up to date]      master     -> origin/master
fatal: couldn't read spotify/.git/packed-refs: Not a directory
```

Cleaning the cache for spotify will probably fix the issue, but I would like to know if you have any idea why this happens.

**Comment by rafasc** [comment]
This indeed looks like an upstream bug. Bisect seems to point to git/git@f57f37e2e1bf11ab4cdfd221ad47e961ba9353a0

From my testing seems like we can work around this by specifying absolute paths for `GIT_DIR` and `GIT_WORK_TREE`.

**Comment by AladW** [comment]
Thanks for the investigation. Happy to merge such a workaround.

**Comment by AladW** [comment]
Merged in #342

**Comment by rafasc** [comment]
The issue is fixed upstream git/git@fb9c2d27039a23457cb9710d86e00c51dfb910dc (> v2.18.0) thus 03a17476ab046961077bde3dad547560e7654f21 can, optionally, be reverted.

Not advocating for a revert, but documenting it here in case we stumble on this in the future.
